doc/source/docker.rst

```diff
@@ -0,0 +1,65 @@
+Docker setup
+============
+
+.. highlight:: bash
+
+
+You can use Docker to run test IOCs that are convenient for testing
+with out having to locally build and install EPICS IOCs.  To communicate
+with the Docker you have set up some environmental variables ::
+
+   #!/usr/bin/bash
+
+   export DOCKER0_IP="172.17.0.1"
+   export EPICS_CA_ADDR_LIST=$( echo $DOCKER0_IP | sed -e 's/^\([0-9]\+\)\.\([0-9]\+\)\..*$/\1.\2.255.255/' )
+   export EPICS_CA_AUTO_ADDR_LIST="no"
+   export EPICS_CA_MAX_ARRAY_BYTES=10000000
+
+
+and to run docker with the correct images (assuming the preceding code
+block is saved in :file:`epics_export`) ::
```

Comment:
Would it be convenient to add such a file to the repo itself? The web community conventionally names this file `env` or `.env`.